### PR TITLE
bandrowski changes

### PR DESCRIPTION
**Pull request by sawyerit**
8/9/17

Chris Bandrowski submission from https://gist.github.com/cbandrow/b07bc28a34a6fabe6e2ec1068d38bb83

### CODE REVIEW

**Comment by sawyerit** [comment]
Cons:
No unit tests
Didn't refactor the error div results to its own method
Some inconsistencies in the quotes
No real comments or docstrings added

Pros: 
let vs var
rename the s var to searchTerm
changed callbacks to => syntax

File: main.js [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
+
+    let url = `https://jira.secondlife.com/activity?maxResults=50&streams=user+IS+${user}&providers=issues`;
+    makeRequest(url, "").then((response) => {
       // empty response type allows the request.responseXML property to be returned in the makeRequest call
+      // without this, more difficult to parse responses that return different formats: JSON vs XML.
       callback(url, response);
     }, errorCallback);
 }
-/**
- * @param {string} searchTerm - Search term for JIRA Query.
```
Comment (chrishague-wf) [inline main.js]: why move the docstrings up to top of file? These seem method specific to me.

File: main.js [hunk 2]
```diff
@@ -39,135 +43,136 @@ function make_request(url, responseType) {
...
-// results.json in the "json_results" folder contains a sample of the API response
-// hint: you may run the application as well if you fix the bug. 
-// 
 
-  return '<p>There may be results, but you must read the response and display them.</p>';
-  
+function createHTMLElementResult(response){
+  document.getElementById('query-result').hidden = false;
+  let resultsArray = response.issues.map((issue) =>{
+    return `<h6>${issue.fields.summary}</h6><ul><li>Status: <strong>${issue.fields.status.name}</strong></li><li>Key: <em>${issue.key}</em>, ID: <em>${issue.id}</em></li><li><a href=${issue.self}>${issue.self}</a></li></ul>`
```
Comment (chrishague-wf) [inline main.js]: any of these fields potentially null?

File: main.js (hunk 1 above)
Comment (toddkaji-wf) [inline main.js]: +1 not sure what went on with this.

**Pull request closed without merging**